sdk/openai/openai-build.yml

```diff
@@ -0,0 +1,78 @@
+steps:
```

Comment:
Honestly, this situation is so unique, to me this is a case where we probably should just have a bespoke pipeline.

Reply:
Yeah. it is unique circumstances.

Reply:
Given the other comment I think I'd rather keep this and have it handle all the setup logic to allow the normal CI matrix be used.